internal/app/handlers/handlers.go

```diff
@@ -0,0 +1,77 @@
+package handlers
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/alexeyvilmost/urlshort.git/internal/app/utils"
+)
+
+type Result struct {
+	Result string `json:"result"`
+}
+
+type Request struct {
+	URL string `json:"url"`
+}
+
+type Handlers struct {
+	BaseURL string
+	Storage map[string]string
+}
+
+func NewHandlers(config *utils.Config) *Handlers {
+	result := new(Handlers)
```

Comment:
result := &Hanlders {

}
```suggestion
	result := &Handlers{
	BaseURL: ...,
	}
```